### ISSUE TITLE
How can I bridge Twilio PSTN Voice Conference to Twilio Video Room?

### ISSUE THREAD

**Issue by bohdandovhan** [twilio/video-quickstart-js#160]
I have read these articles
https://www.twilio.com/docs/voice/conference
https://www.twilio.com/docs/video/tutorials/understanding-video-rooms
https://www.twilio.com/blog/now-you-can-add-programmable-voice-participants-video-group-rooms
but I still struggle to understand how can I bridge Twilio PSTN Voice Conference to Twilio Video Room?

I see the information in the documentation that

> The \<Connect\> verb is designed to connect individual PSTN phone calls into a Video Room. This functionality should not be used to bridge a Programmable Voice Conference with a Video Room. This is an unsupported use case and it can fail in unexpected ways.

So I guess, I can join individual participants to Twilio Video Room.

However, I need them wait until the host starts Twilio Video Room.

Is this possible to achieve?

To summarize, I have two problems.
1. Is there any better way to bridge a Programmable Voice Conference with a Video Room than connecting every individual PSTN caller through \<connect\> verb?
2. How can I make PSTN callers wait until the host starts the Twilio Video Room?

**Comment by bohdandovhan** [comment]
I have found an option to make PSTN Callers wait until the host starts the Twilio Video Room.

When PSTN Participant joins, TwiML app should return a Say command with a waiting message with an infinite loop (loop = 0), which will repeat the message 1000 times.

Then Update a Call Resource can be used either to terminate the call by timeout or execute the '<connect>' verb to connect the participant to the '<room>'
